src/core/graph.test.js

```diff
@@ -1663,4 +1664,87 @@ describe("core/graph", () => {
       expect(edgeToParts(edge)).toEqual(expected);
     });
   });
+
+  describe("compareGraphs", () => {
+    describe("equivalent graphs", () => {
+      const graph1 = new Graph().addNode(src).addNode(dst).addEdge(simpleEdge);
+      const graph2 = new Graph().addNode(src).addNode(dst).addEdge(simpleEdge);
```

Comment:
These two graphs are very simple (e.g. they do not include dangling edges) and also have the exact same creation history. It would be better to stress test, e.g.:

graphs that have dangling edges
two graphs which are the same but have very different history (e.g. nodes that were added and removed)

Happily, we have helper methods for doing so:
https://github.com/sourcecred/sourcecred/blob/master/src/core/graphTestUtil.js

Reply:
done